elasticdl/pkg/commonnew/embedding_tabel.go

```diff
@@ -0,0 +1,70 @@
+package commonnew
+
+import (
+	"elasticdl.org/elasticdl/pkg/proto"
+	"github.com/tensorflow/tensorflow/tensorflow/go/core/framework/types_go_proto"
+)
+
+// EmbeddingTable struct
+type EmbeddingTable struct {
```

Comment:
We need a function to serialize a `EmbeddingTable` instance to a `proto.IndexedSlices`. It will be used when making checkpoint in PS.

Reply:
added to tensor.go

Reply:
for most of the time, we don't need to serialize all the data inside `EmbeddingTable` to an indxedSlice.
So we could use like this:
```
//given: indices []int64, table *EmbeddingTable
tensor := table.GetEmbeddingVectors(indices)
tensor.SerializerToIndexedSlice(indices)
```
`GetEmbeddingVectors` funtion will be independently used in optimizer.